static/src/javascripts/projects/common/modules/userFeatures/cookies/adFree.ts

```diff
@@ -0,0 +1,11 @@
+import { getCookie } from '@guardian/libs';
+import { userBenefitsDataIsUpToDate } from './userBenefitsExpiry';
+
+export const AD_FREE_USER_COOKIE = 'GU_AF1';
+
+export const adFreeDataIsPresent = (isSignedIn: boolean): boolean =>
```

Comment:
This logic is described here: https://docs.google.com/document/d/19ExTKgMhCJbrSLD9GFTeNKy3ArjoS2GR8wVc2ONbNXo/edit?tab=t.0#heading=h.kz4jiwdknjbq